doc/operation-and-maintenance/Cluster-management-considerations.md

```diff
@@ -72,3 +72,24 @@ Scalability highly depends on variables such as:
     * Presences
     * HTTP notifications (with queuing systems such as RabbitMQ or Kafka)
 * latency of messaging, both real-time and archived messages
+
+#### OS configuration
+
+To achieve high scalability you have to adjust configuration of your operating system.
+
+First, set some network related parameters - this is what we use for load testing:
+
+Parameter | Value
+----|----
+net.ipv4.ip_local_port_range | 1024 65535
+net.ipv4.tcp_mem | 16777216 16777216 16777216
+net.ipv4.tcp_wmem | 4096 87380 16777216
+net.ipv4.tcp_rmem | 4096 87380 16777216
+
+Then, you have to increase the number of open files allowed for the user running your mongooseim server process.
+In Linux, this is most commonly done in `/etc/security/limits.conf` file.
+You should remember, though, that there is a limit to it - you can't increase it above an upper bound which is set by a kernel param `fs.file-max`. 
```

Comment:
```suggestion
You should remember, though, that there is a limit to it — you can't increase it above an upper bound which is set by the `fs.file-max` kernel parameter.
```
(definitely the hyphen should be an em-dash here, GH on console fonts doesn't show the difference between the three - – —, but readthedocs will)

Or rewording things, but this is very opinionated:
```suggestion
Notice though that this has an upper limit configured elsewhere: you can't increase it above the `fs.file-max` kernel parameter. 
```